### DISCUSSION TITLE
Difference between 2 dates in Enhanced Table

### DISCUSSION THREAD

**Discussion by Sandy381** [fbaligand/kibana-enhanced-table#296] [General]
Hi,
 
I am using Enhanced table to show difference between integer numbers of 2 columns.

I was wondering If it is possible to show difference between 2 dates in anyway.

Eg:
Col1 = "Nov 8, 2022" 
Col2 = "Nov 10, 2022" 

Col2 - Col1 = 2 days.

Any ideas?

**Comment by fbaligand** [comment]
Well, if Col1 and Col2 are really date columns, the value should be milliseconds count since Epoch.
So you can create a computed column with:
- Formula: `(col2 - col1) / 1000 / 60 / 60 / 24`
- 'Apply template' checked
- Template: `{{value}} days`

**Reply by Sandy381** [reply]
Hi @fbaligand ,

In another index, I'm using  Date values in  ISO-8601 format: 'created' : '2022-12-10T17:10:14.203Z' , 'updated' : '2022-12-12T17:20:56.409Z'.
Any way can i achieve the difference of 2 dates?

**Reply by fbaligand** [reply]
Yes, you can do it using this formula:
`parseDate(col['updated']) - parseDate(col['created'])`

It gives you the number of milliseconds between the two dates.
